test/components/BaseForm_test.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { fireEvent, render } from "@testing-library/react";
+import BaseForm from "../../src/components/BaseForm";
+
+const testSchema = {
+  type: "object",
+  properties: {
+    title: {
+      type: "string",
+      title: "Title",
+      description: "Short title",
+    },
+    content: {
+      type: "string",
+      title: "Content",
+      description: "Provide details...",
+    },
+  },
+};
+
+const testUiSchema = {
+  "ui:order": ["title", "content"],
+  content: {
+    "ui:widget": "textarea",
+  },
+  id: {
+    "ui:widget": "text",
+    "ui:disabled": true,
+  },
+  last_modified: {
+    "ui:widget": "hidden",
+    "ui:disabled": true,
+  },
+  schema: {
+    "ui:widget": "hidden",
+    "ui:disabled": true,
+  },
+  "ui:disabled": false,
+};
+
+describe("BaseForm component", () => {
+  beforeEach(() => {});
+
+  afterEach(() => {});
```

Comment:
Do we need these?

Reply:
Nope! Added them out of habit and forgot to remove them because prettier made them too small. I noticed them in the next branch and will be removing them then.